compair/static/modules/login/login-directive.js

```diff
@@ -33,6 +33,7 @@
                         uses_compair_login: scope.uses_compair_login,
                         email_notification_method: EmailNotificationMethod.enable
                     }
+                    scope.loggedInUserIsStudent = true;
```

Comment:
Good question. 

This happens when a user who currently doesn't have a ComPAIR account (could be student or instructor) is linking their LTI or CAS/SAML that they just logged in with to a new ComPAIR account.

I set the default to student to be safe. A little lower it is being set again based on the system role `LTI.getLTIUser();` returns (this will update an instructor's and re-enable the fields for them).

In the second part of this issue #734 I have the disabled student fields automatically refresh when a student logins (so it would be overwritten when they next login). 

I should have added some of that info to the commit message/PR